core/.js/src/test/scala/io/udash/routing/RoutingEngineTest.scala

```diff
@@ -37,7 +38,7 @@ class RoutingEngineTest extends UdashFrontendTest with TestRouting {
         }
       }
 
-      initTestRoutingEngine(state2vp = state2VP)
+      initTestRoutingEngine(state2vp = state2VP.view.mapValues(() => _).toMap)
```

Comment:
Nit (applies to the line 296 too): is using `view` and `toMap` necessary if there is only one operation to be executed on map values?

Reply:
No `mapValues` in Scala 2.13